step/step.go

```diff
@@ -64,7 +71,13 @@ func (e EnvVarSharer) ProcessConfig() (*Config, error) {
 	stepconf.Print(input)
 	e.logger.Println()
 
-	envVars, err := e.parseEnvVars(input.EnvVars)
+	secretKeys := e.secretKeysProvider.Load(e.envRepository)
+
+	if len(secretKeys) == 0 {
+		e.logger.Debugf("Secret keys list is empty.")
```

Comment:
Ahh thanks for telling me this. I will convert it to a info message then. I could have removed it too but I think this will help with future SWAT issues.

Reply:
What about a simple Print? because I feel like this is not as important as for what we usually use the info level.

Reply:
Good point. I have changed it to a print. 👍🏻